src/components/LoginDialog.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <q-card style="min-width: 25%">
+    <q-card-section>
+      <div class="text-h6">
+        Log in
+      </div>
+    </q-card-section>
+    <form @submit.prevent="onSubmit">
+      <q-card-section>
+        <q-input
+          v-model="username"
+          dense
+          outlined
+          placeholder="Username"
+          :stack-label="true"
```

Comment:
This should be removed I think. It only works in conjunction with a label and you're using a placeholder instead.

Reply:
sorry, will do!